nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -188,7 +188,8 @@ def input_types(self):
         """Returns definitions of module input ports."""
         return OrderedDict(
             {
-                "audio_signal": NeuralType(('B', 'D', 'T'), SpectrogramType()),
+                # to handle we stack one ConformerEncoder on the other
+                "audio_signal": NeuralType(('B', 'D', 'T'), ChannelType()),
```

Comment:
Why do you need to change the typing here?

Reply:
that original FC takes mel feature as input. Here we want to allow stacking 2 more FC on top of the original pretrained FC encoder as the modality adapter. Hence try to relax it to  ChannelType